app/controllers/api/recipes_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::RecipesController < ApplicationController
+  def index
+    recipes = Recipe.all
```

Comment:
Did we agree that we are gonna return the latest recipes?
Also a limit should be in place, like the last 30-20 recipes

Reply:
Yes we did! I thought it might be out of scope, but let's do it now.